.circleci/config.yml

```diff
@@ -16,7 +16,502 @@ aliases:
+      - checkout
+      - run:
+          name: CI job started
+          command: ./scripts/ci-job-started.sh
+      - run:
+          name: Copy artifacts from workspace
+          command: cp -r /tmp/workspace/oss/* .
+      - run:
+          name: Package Grafana
+          command: /tmp/workspace/bin/grabpl package
+      - run:
+          name: Move artifacts
+          command: |
+            mkdir -p oss
+            mv dist oss/
+      - persist_to_workspace:
+          root: .
+          paths:
+            - oss/dist/*
+      - run:
+          name: CI job failed
+          command: ./scripts/ci-job-failed.sh
+          when: on_fail
+      - run:
+          name: CI job succeeded
+          command: ./scripts/ci-job-succeeded.sh
+          when: on_success
+
+  package-enterprise:
+    executor: grafana-build
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - checkout
+      - run:
+          name: CI job started
+          command: ./scripts/ci-job-started.sh
+      - run:
+          name: Copy artifacts from workspace
+          command: cp -r /tmp/workspace/enterprise/* .
+      - run:
+          name: Package Grafana
+          command: /tmp/workspace/bin/grabpl package --enterprise
+      - run:
+          name: Move artifacts
+          command: |
+            mkdir -p enterprise
+            mv dist enterprise/
+      - persist_to_workspace:
+          root: .
+          paths:
+            - enterprise/dist/*
+      - run:
+          name: CI job failed
+          command: ./scripts/ci-job-failed.sh
+          when: on_fail
+      - run:
+          name: CI job succeeded
+          command: ./scripts/ci-job-succeeded.sh
+          when: on_success
+
+  publish-oss-master:
+    executor: grafana-publish
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - checkout
+      - run:
+          name: CI job started
+          command: ./scripts/ci-job-started.sh
+      - run:
+          name: Trigger Windows build
+          command: ./scripts/trigger_windows_build.sh ${APPVEYOR_TOKEN} ${CIRCLE_SHA1} master
+      - run:
+          name: GCP credentials
+          command: echo ${GCP_GRAFANA_UPLOAD_KEY} > /tmp/gcpkey.json
+      - run:
+          name: Sign in to GCP
+          command: /opt/google-cloud-sdk/bin/gcloud auth activate-service-account --key-file=/tmp/gcpkey.json
+      - run:
+          name: Publish to GCP
+          command: |
+            echo /opt/google-cloud-sdk/bin/gsutil cp /tmp/workspace/oss/dist/* gs://$GCP_BUCKET_NAME/oss/master
+            # /opt/google-cloud-sdk/bin/gsutil cp /tmp/workspace/oss/dist/* gs://$GCP_BUCKET_NAME/oss/master
+      - run:
+          name: Publish to Grafana.com
+          command: |
+            cp -r /tmp/workspace/oss/dist .
+            rm -f dist/*latest*
+            cd dist
+            echo ../scripts/build/release_publisher/release_publisher -apikey ${GRAFANA_COM_API_KEY} -version "v$(cat grafana.version)" --nightly
```

Comment:
I don't think this will end up echoing the grafana api key to the logs (I think CircleCI will protect us from that) but it might still be a good idea not to write it like this as it looks like that is what will happen.

Reply:
@xlson Oh yeah, you're right. CircleCI does hide API keys I'm pretty sure, but I shouldn't echo it either. I'll remove it. Thanks!